### ISSUE TITLE
Deep Learning-based Codebook Design for Code-domain Non-Orthogonal Multiple Access Achieving a Single-User Bit Error Rate Performance. (arXiv:2104.00818v1 [cs.IT])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#5518]
Deep Learning-based Codebook Design for Code-domain Non-Orthogonal Multiple Access Achieving a Single-User Bit Error Rate Performance. (arXiv:2104.00818v1 [cs.IT])<br>
https://ift.tt/3mn78Xc<br>
<p>The codebook design for code-domain non-orthogonal multiple access (CD-NOMA) can be considered as a constellation design for multi-user multi-dimensional modulation (MU-MDM). This paper proposes an autoencoder (AE)-based constellation design for MU-MDM with the objective of achieving a comparable bit error rate (BER) performance to single-user multi-dimensional modulation (SU-MDM), i.e., alleviating performance degradation in non-optimal AE design caused by overloading multiple users. Recognizing that various constraints in a receiver structure degrade a BER performance of the codebook design in the existing CD-NOMA, the MU-MDM design aims at global optimization on the common ground with SU-MDM by leveraging the agnosticism of the neural network-based multi-user decoder obtained from AE training, while mitigating the power normalization constraint and exploiting dense resource mapping in the MU-MDM AE structure. Moreover, as opposed to the existing loss function for MU-MDM which has failed to minimize BER for different levels of signal-to-noise ratio (SNR), a hyperparameterized loss function and proper training procedures are introduced to jointly optimize the signal points for MU-MDM constellation and their bit-to-symbol mapping. It has been demonstrated that the proposed design achieves a single-user BER bound with only 0.2dB loss, equivalently outperforming the existing CD-NOMA designs, while maintaining their overloading factor.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
